### DISCUSSION TITLE
Manual creation of an entire train itinerary, regardless of where you get on and off the train

### DISCUSSION THREAD

**Discussion by pascalwittler** [Traewelling/traewelling#2652] [Ideas / Feature Requests]
Today's use case for creating a manual trip was an ICE train that was scheduled to end in Basel (for whatever reason), but actually went to Frankfurt. [1] The partial journey from Interlaken Ost to Basel SBB was already available for check-in. [2], [3], [4] So I wanted to create a manual trip for the missing part from Basel SBB to Frankfurt (Main) Hbf and then check in from Karlsruhe Hbf to Frankfurt (Main) Hbf. [5]

However, after filling in the stops, arrival and departure times into the form [5], the dialog for selecting the station where you get off the train appeared. The stop where you get on could not be selected [6]. I could only check in for the entire trip from Basel SBB to any stop in the list (in my case Frankfurt).

Since I did not want to do that, I closed the exit stop selection popup with the × button but then all the data entered earlier disappeared and I ended up back in the normal check-in dialog (also with an error message because neither the stop nor the time had been set). [7]

So my feature request is the ability to create an entire train itinerary, regardless of where you get on and off the train, and then check in using the newly created train from the departure list.

[1] The actual train itinerary of ICE 376, including the additional stops with green text color

![1](https://github.com/Traewelling/traewelling/assets/20669380/38de3e2a-7c33-4e97-ad1f-b652417b11fc)

[2] The list of available trains to check in from Interlaken Ost

![2](https://github.com/Traewelling/traewelling/assets/20669380/d1821f37-2ddf-4723-bcdd-e23e764bdab2)

[3] The selectable train journey in Träwelling according to the schedule

![3](https://github.com/Traewelling/traewelling/assets/20669380/25394338-cdfc-4f3d-b0e9-0905331ba74f)

[4] ICE 376 missing in Karlsruhe at its re-scheduled departure

![4](https://github.com/Traewelling/traewelling/assets/20669380/cbbc85a7-6e1a-4b7b-8ddf-f305a5b15769)

[5] The filled in manual train trip creation form

![5](https://github.com/Traewelling/traewelling/assets/20669380/9e619d63-ba2d-4721-b540-3d8904fbfb62)

[6] The dialog to choose where to get off the train after clicking the Save button

![6](https://github.com/Traewelling/traewelling/assets/20669380/a1139ec4-cca3-4d5b-89dd-7510003a12b7)

[7] The "lost place" after closing the popup

![7](https://github.com/Traewelling/traewelling/assets/20669380/ed870f74-d127-45d2-9e58-b6880c485cce)

**Comment by MrKrisKrisu** [comment]
This is now possible with the new creation process in the experimental features.

**Comment by MrKrisKrisu** [comment]
This is now possible with the new creation process in the experimental features.
